scripts/windows/commit.txt

```diff
@@ -0,0 +1 @@
+480deec8b5dde3cd331d1a793106981c1796d172
```

Comment:
My intention is to have this hash shared with `pyspy_build.sh`. The PR doesn't do it - it's still cloned. Please modify `pyspy_build.sh` as well, to use the commit & tag files. You can place both in `scripts/pyspy_commit` and `scripts/pyspy_tag`.